docs/faq.rst

```diff
@@ -344,17 +344,6 @@ when you perform a delete, instead, a marker (also called a "tombstone") is writ
 Never fear though, on the first compaction that occurs between the data and the tombstone, the data will be expunged
 completely and the corresponding disk space recovered. 
 
-What are seeds?
-^^^^^^^^^^^^^^^
-
-Seeds are used during startup to discover the cluster. They are referred by new nodes on bootstrap to learn about other nodes in the ring. When you add a new node to the cluster, you
-must specify one live seed to contact.
```

Comment:
AFAIK, this part is still true.
We do have seeds in scylla.yaml, so we may as well document them somewhere.